topics/spacevim/index.md

```diff
@@ -0,0 +1,16 @@
+---
+aliases: space vim
+created_by: ShiDong Wang
+display_name: SpaceVim
+github_url: https://github.com/SpaceVim/SpaceVim
+logo: spacevim.png
+released: November 6, 2017
```

Comment:
On https://github.com/SpaceVim/SpaceVim/releases, the earliest release, 0.1.0, came out January 26, 2017.

Reply:
So we nee to use the first release? ok, will change this.